documentation/src/test/java/org/hibernate/userguide/pc/FilterJoinTableTest.java

```diff
@@ -118,10 +115,6 @@ public void testLifecycle() {
             type="int"
         )
     )
-    @Filter(
-        name="firstAccounts",
-        condition="order_id <= :maxOrderId"
-    )
```

Comment:
The above filter is not used for `Client` but for its `accounts` collection field. Seems an overlooking and might be misleading.